api.js

```diff
@@ -286,7 +185,7 @@ const api = {
             to: channelId,
             message: `<@${userId}> ${message}`
           });
-          let glimmer = (Number(snapshot.val().glimmer) + Number(newAmount));
+          let glimmer = (Number(snapshot.val().glimmer) + Number(newAmount)) < 0 ? 0 : Number(snapshot.val().glimmer) + Number(newAmount);
```

Comment:
You can see on the left side I changed this to let them go negative, just make sure there are no differences here after you pull